internal/schemautil/service.go

```diff
@@ -699,6 +700,9 @@ func FlattenServiceComponents(r *aiven.Service) []map[string]interface{} {
 			"route":                       c.Route,
 			"usage":                       c.Usage,
 			"kafka_authentication_method": c.KafkaAuthenticationMethod,
+			// By default, endpoints are always encrypted and
+			// this property is only included for service components that may disable encryption.
+			"ssl": PointerValueOrDefault(c.Ssl, true),
```

Comment:
We should always propagate what is coming from the API response, and often this field is not set in the response, and in these situations, we keep it empty in Terraform state.

Reply:
It is in a state anyway, but with with a wrong value:

```json
"ssl": false,
```

And since it is computed, we can't set it's default value.  
You can also see it in the [issue](https://github.com/aiven/terraform-provider-aiven/issues/992).

Reply:
Then I think we can set `null` https://developer.hashicorp.com/terraform/language/expressions/types#null to this field or, in the worst case scenario, change a type to a string and when it is not set, set it empty.

Reply:
Hey, I did some experiments with the `null` value and it is not that simple. So I guess it is a good idea in the end to set it `true`